doc/rados/operations/erasure-code.rst

```diff
@@ -173,13 +173,13 @@ data in an erasure coded pool:
 
     ceph osd pool set ec_pool allow_ec_overwrites true
 
-This can only be enabled on a pool residing on bluestore OSDs, since
-bluestore's checksumming is used to detect bitrot or other corruption
-during deep-scrub. In addition to being unsafe, using filestore with
-ec overwrites yields low performance compared to bluestore.
+This can only be enabled on a pool residing on BlueStore OSDs, since
```

Comment:
As long as I'm starting a review, I might as well suggest this:

s/This can only be enabled on a pool/This can be enabled only on a pool/